resources/ansible/roles/uploaders/templates/upload-random-data.sh.j2

```diff
@@ -1,5 +1,33 @@
 #!/usr/bin/env bash
 
+# Check if SECRET_KEY is set and not empty, then append to .bashrc if not already present
```

Comment:
Can you please remove these comments from the script? The code says this fine on its own.